src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -402,6 +402,7 @@
                                           IsChecked="{x:Bind IsBellStyleFlagSet(4), BindBack=SetBellStyleTaskbar, Mode=TwoWay}" />
                             </StackPanel>
                         </local:SettingContainer>
+

```

Comment:
Ah, I see you thought you had to edit Profiles.xaml haha. You can definitely undo this one.